### PR TITLE
Add release notes for 8.0.0

### PR DESCRIPTION
**Pull request by melissalinkert**
Preparation for tomorrow's release of 8.0.0.

### CODE REVIEW

File: sphinx/about/whats-new.rst [hunk 1]
```diff
@@ -1,6 +1,72 @@
...
+* add documentation for new batch processing feature in `showinf` (thanks to Torsten Stöter)
+* add documentation for precompressed tiles feature [1]_
+* several other fixes for broken links
+
+Component updates:
+
+* `ome-common` was upgraded to 6.0.24
+* `cdm-core` was upgraded to 5.6.0
+
+.. [1] This functionality was implemented through collaboration between Glencoe Software and `NCI Imaging Data Commons <https://datacommons.cancer.gov/repository/imaging-data-commons/>`_,
```
Comment (sbesson) [inline sphinx/about/whats-new.rst:66]: Really like the format. Only wonder is whether the use of numbers  as the reference scales well. Last thing we want to do is reindex all of these everytime we add a new footnote.

File: sphinx/about/whats-new.rst [hunk 2]
```diff
@@ -1,6 +1,72 @@
...
+* ImageJ export plugin: fix handling of multi-channel datasets
+* `JPEGTurboService` [1]_
+   - add several new methods for retrieving tiles from a single JPEG stream
+   - expand Javadocs for existing methods
+
+Documentation updates:
+
+* update all links to AwareSystems TIFF documentation to last version from Internet Archive
+    - see `libtiff mailing list discussion <https://www.mail-archive.com/tiff@lists.osgeo.org/msg00421.html>`_ for context
+* fix index of comannd-line tools
```
Comment (sbesson) [inline sphinx/about/whats-new.rst:56]:
```suggestion
* fix index of command-line tools
```

File: sphinx/about/whats-new.rst [hunk 3]
```diff
@@ -1,6 +1,72 @@
...
+   - add support for writing precompressed tiles [1]_
+
+* OME-XML
+   - fix incorrect logical operators when checking image dimensions
+
+* RCPNL
+   - define objective metadata for lens ID 18112
+
+* SDT
+   - fix reading of files acquired with SPCM 9.90 (thanks to Chen Qian)
```
Comment (jburel) [inline sphinx/about/whats-new.rst:37]: i assume Chen Qian will be added to the list of contributors on www.o.org

Comment (melissalinkert) [reply sphinx/about/whats-new.rst:37]: Yes, that is done in https://github.com/ome/www.openmicroscopy.org/pull/757

**Review by sbesson — APPROVED** [review]

**Pull request merged**